druid/examples/game_of_life.rs

```diff
@@ -12,125 +12,35 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#![allow(clippy::unreadable_literal)]
-
-//! Game of life
+//! This is an example of how you would implement the game of life with druid.
+//! This example doesnt showcase anything specific in druid.
 
 use std::ops::{Index, IndexMut};
 use std::time::{Duration, Instant};
 
 use druid::widget::prelude::*;
 use druid::widget::{Button, Flex, Label, Slider};
 use druid::{
-    AppLauncher, Color, Data, Lens, LocalizedString, MouseButton, Point, Rect, TimerToken,
-    WidgetExt, WindowDesc,
+    AppLauncher, Color, Data, Lens, MouseButton, Point, Rect, TimerToken, WidgetExt, WindowDesc,
 };
 use std::sync::Arc;
 
-const GRID_SIZE: usize = 40;
+const GRID_SIZE: usize = 41;
```

Comment:
what's this about?

Reply:
This is so that the background grid is not aligned. If it is aligned you get straight lines as cels which is a bit boring, and also doesn't highlight that it is actually a grid. This makes it so that neighboring cells (almost?? I dont know for sure anymore) always have a different colour.